src/libs/actions/Session.js

```diff
@@ -376,22 +375,6 @@ function clearAccountMessages() {
     Onyx.merge(ONYXKEYS.ACCOUNT, {error: '', success: ''});
 }
 
-/**
```

Comment:
Overall this is the only part I'm iffy about – I'm not sure it's worth moving these to their own actions just to prevent the dependency cycle (that I _think_ would be safe). Could we prevent the dependency cycle by moving them to `src/libs/actions/Session/setSessionLoadingAndError`, then importing & exporting them in `src/libs/actions/Session/index.js` ?

Reply:
Yeah I agree. They do feel a little disconnected now.